app/components/provider_interface/application_summary_component.rb

```diff
@@ -43,6 +52,13 @@ def support_reference_row
       }
     end
 
+    def application_number_row
+      {
+        key: 'Application number',
+        value: @application_choice.id,
```

Comment:
could we add `application_choice` as an `attr_reader` and avoid setting the `@`?